front_end/entrypoints/main/MainImpl.ts

```diff
@@ -313,12 +313,6 @@ export class MainImpl {
         'timelineAsConsoleProfileResultPanel', 'View console.profile() results in the Performance panel for Node.js',
         true);
 
-    // JS Profiler
-    Root.Runtime.experiments.register(
-        'jsProfilerTemporarilyEnable', 'Enable JavaScript Profiler temporarily', /* unstable= */ false,
-        'https://developer.chrome.com/blog/js-profiler-deprecation/',
-        'https://bugs.chromium.org/p/chromium/issues/detail?id=1354548');
-
```

Comment:
This is removed, and replicated inside `rn_inspector.ts`. This is to register the experiment in time to enable it in that context — otherwise we're unable to cleanly wait for `MainImpl` to initialise (without other changes).